ansible/roles/git_lfs/tasks/main.yaml

```
@@ -0,0 +1,17 @@
+# https://github.com/git-lfs/git-lfs/wiki/Installation#debian-and-ubuntu
+- name: Download script
+  ansible.builtin.get_url:
+    url: https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh
+    dest: /tmp/script.sh
+    mode: 0764
+- name: Setup
+  become: true
+  environment:
+    os: "{{ ansible_distribution }}"
+    dist: "{{ ansible_distribution_release }}"
+  ansible.builtin.script: /tmp/script.sh
+- name: Install Git LFS
```

[style] Added empty lines.

```suggestion

- name: Setup
  become: true
  environment:
    os: "{{ ansible_distribution }}"
    dist: "{{ ansible_distribution_release }}"
  ansible.builtin.script: /tmp/script.sh

- name: Install Git LFS
```